src/components/common/Item.js

```diff
@@ -73,7 +73,7 @@ const Item = ({ id, isChildren, showPinnedOnly }) => {
     return <Alert severity="error">{t('An unexpected error occured.')}</Alert>;
   }
 
-  const showCollapse = item.get('settings')?.isExpandable;
+  const showCollapse = item.get('settings')?.isCollapsible;
```

Comment:
I think we did well in renaming those, because even your branch name is confusing 😆